cmd/root.go

```diff
@@ -227,13 +245,9 @@ func sendMessageToURL(msg string, queue Queue) bool {
 		}
 	}
 
-	for {
-		resp, err = httpClient.Post(endpoint, "application/json", bytes.NewBuffer([]byte(msg)))
-		if err == nil {
-			break
-		}
-		log.Printf("%s: Error hitting endpoint with msg '%s', retrying after 1 second... Error: %s", queue.Name, msg, err.Error())
-		time.Sleep(time.Second)
+	resp, err = httpClient.Post(endpoint, "application/json", bytes.NewBuffer([]byte(msg)))
+	if err != nil {
+		log.Printf("%s: Error hitting endpoint with msg '%s'... Error: %s", queue.Name, msg, err.Error())
```

Comment:
Should this be conditional to support backward compatibility?
I meant
```
if healthCheckURL 
   don't retry
else 
   keep retrying
```

Reply:
@ameykpatil Makes sense. Ideally, we should always use a healthcheck url, but lets not exhaust the retries if someone don't configure it.
I'll change this 👍 

P.S.: Should we make healthcheck url compulsory?